client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -96,12 +101,12 @@ const Checkout = React.createClass( {
 	},
 
 	addProductToCart: function() {
-		var planSlug = this.props.plans.getSlugFromPath( this.props.product ),
+		var planSlug = getUpgradePlanSlugFromPath( this.props.product ),
```

Comment:
I agree, It's probably best to handle that in a separate janitorial PR if only to keep the diff down on this PR.